test/e2e/build_templates.go

```diff
@@ -305,40 +305,51 @@ type TaskRunData struct {
 	Status string
 }
 
-func ListAllTaskRunsOutput(t *testing.T, cs *Clients, td map[int]interface{}) string {
-
-	const (
-		emptyMsg = "No taskruns found"
-		header   = "NAME\tSTARTED\tDURATION\tSTATUS\t"
-		body     = "%s\t%s\t%s\t%s\t\n"
-	)
+func ListAllTaskRunsOutput(t *testing.T, cs *Clients, allnamespaces bool, td map[int]interface{}) string {
 
+	const listTemplate = `{{- $trl := len .TaskRuns.Items }}{{ if eq $trl 0 -}}
+No TaskRuns found
+{{- else -}}
+NAME	STARTED	DURATION	STATUS{{- if $.AllNamespaces }}	NAMESPACE{{- end }}
+{{- range $_, $tr := .TaskRuns.Items }}
+{{- if $tr }}
+{{ $tr.Name }}	{{ formatAge $tr.Status.StartTime $.Time }}	{{ formatDuration $tr.Status.StartTime $tr.Status.CompletionTime }}	{{ formatCondition $tr.Status.Conditions }}{{- if $.AllNamespaces }}	{{ $tr.Namespace }}{{- end }}
```

Comment:
Same comment as for PipelineRuns:

I'm assuming that `{{- if $.AllNamespaces }}` is throwing off the spacing for the `NAMESPACE` header here. When I run `tkn tr ls -A` locally, this is what I get:

```
NAME                STARTED        DURATION     STATUS               NAMESPACE
sleep1s-run-k6kxd   22 hours ago   17 seconds   Succeeded   default
sleep1s-run-4fd8q   22 hours ago   20 seconds   Succeeded   default
sleep1s-run-j2vwf   22 hours ago   16 seconds   Succeeded   default

```